pkg/liqo-controller-manager/external-network/configuration-controller/configuration-controller.go

```diff
@@ -82,6 +97,25 @@ func (r *ConfigurationReconciler) Reconcile(ctx context.Context, req ctrl.Reques
 	return ctrl.Result{}, err
 }
 
+func (r *ConfigurationReconciler) defaultLocalNetwork(ctx context.Context, cfg *networkingv1alpha1.Configuration) error {
+	if r.localCIDR == nil {
+		ipamStorage, err := liqogetters.GetIPAMStorageByLabel(ctx, r.Client, labels.NewSelector())
+		if err != nil {
+			return fmt.Errorf("unable to get IPAM storage: %w", err)
+		}
+
+		r.localCIDR = &networkingv1alpha1.ClusterConfigCIDR{
+			Pod:      networkingv1alpha1.CIDR(ipamStorage.Spec.PodCIDR),
+			External: networkingv1alpha1.CIDR(ipamStorage.Spec.ExternalCIDR),
+		}
+	}
+
+	cfg.Spec.Local = &networkingv1alpha1.ClusterConfig{
+		CIDR: *r.localCIDR,
+	}
+	return r.Client.Update(ctx, cfg)
+}
```

Comment:
It can cause problems if we change the ipam storage CRD with a database. We surelly need to insert these information in a configmap or to pass it to the controller-manager with a flag (especially if we want to move the IPAM from the network-manager to the controller-manager).

We can fix it in a next PR, when we will migrate the IPAM